include/range/v3/utility/inline_variable.hpp

```diff
@@ -0,0 +1,38 @@
+// Range v3 library
+//
+//  Copyright Eric Niebler 2013-2014
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/ericniebler/range-v3
+//
+
+#ifndef RANGES_V3_UTILITY_INLINE_VARIABLE_HPP
+#define RANGES_V3_UTILITY_INLINE_VARIABLE_HPP
+
+#include <range/v3/range_fwd.hpp>
+
+#if RANGES_CXX_INLINE_VARIABLES < RANGES_CXX_INLINE_VARIABLES_17
+namespace ranges
+{
+    inline namespace v3
+    {
+        namespace detail
+        {
+            template<typename T>
+            struct inline_variable_
+            {
+                static constexpr T value {};
+            };
+
+            template<typename T>
+            constexpr T inline_variable_<T>::value;
+        }
+    }
+}
+#endif  // RANGES_CXX_INLINE_VARIABLES
+
+#endif
```

Comment:
This is a utility for building static const objects, not inline variables. The name was right before.